__tests__/frozen.js

```diff
@@ -0,0 +1,49 @@
+"use strict"
+import immer from ".."
+
+describe("auto freeze", () => {
+    const baseState = {
+        object: { a: 1 },
+        array: [1, 2]
+    }
+
+    it("should freeze objects after modifications", () => {
+        expect(Object.isFrozen(baseState.object)).toBe(false) // initially not frozen
+        const next = immer(baseState, draft => {
+            draft.object.c = 2
+        })
+        expect(Object.isFrozen(next.object)).toBe(true)
+        expect(Object.isFrozen(next)).toBe(true)
```

Comment:
Should we also safely assert that `next.array` is not forzen in the process ?

Reply:
Done